cmd/blockchaincmd/convert.go

```diff
@@ -395,7 +389,7 @@ func InitializeValidatorManager(
 		return tracked, err
 	}
 	// TODO: replace latest below with sig agg version in flags for convert and deploy
-	err = signatureaggregator.CreateSignatureAggregatorInstance(app, subnetID.String(), network, extraAggregatorPeers, aggregatorLogger, "latest")
+	err = signatureaggregator.CreateSignatureAggregatorInstance(app, network, aggregatorLogger, "latest")
```

Comment:
if you decide to encapsulate creation inside endpoint retrieval, then remove this